x/cdp/keeper/auctions.go

```diff
@@ -109,7 +109,12 @@ func (k Keeper) RunSurplusAndDebtAuctions(ctx sdk.Context) error {
 	remainingDebt := k.GetTotalDebt(ctx, types.LiquidatorMacc)
 	params := k.GetParams(ctx)
 	if remainingDebt.GTE(params.DebtAuctionThreshold) {
-		_, err := k.auctionKeeper.StartDebtAuction(ctx, types.LiquidatorMacc, sdk.NewCoin("usdx", remainingDebt), sdk.NewCoin(k.GetGovDenom(ctx), remainingDebt.Mul(sdk.NewInt(dump))), sdk.NewCoin(k.GetDebtDenom(ctx), remainingDebt))
+		debtLot := sdk.NewCoin(k.GetDebtDenom(ctx), params.DebtAuctionLot)
+		// debtAuctionLot * remainingDebt / remainingDebt
```

Comment:
q: why are we multiplying and dividing by `remainingDebt` instead of just using `debtAuctionLot`?

Reply:
😂 this formula is totally unnecessary, will remove